iocage_cli/import.py

```diff
@@ -21,16 +22,47 @@
 # STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
 # IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
-"""import module for the cli."""
+"""Export a jail from the CLI."""
 import click
 
-import iocage_lib.iocage as ioc
+import iocage.errors
+import iocage.Jail
+import iocage.Host
+import iocage.ZFS
+
+from .shared.click import IocageClickContext
 
 __rootcmd__ = True
 
 
-@click.command(name="import", help="Import a specified jail.")
+@click.command(name="import", help="Import a jail from a backup archive")
+@click.pass_context
 @click.argument("jail", required=True)
-def cli(jail):
-    """Import from an iocage export."""
-    ioc.IOCage(jail=jail).import_()
+@click.argument("source", required=True)
+def cli(
+    ctx: IocageClickContext,
+    jail: str,
+    source: str
+) -> None:
+    """Restore a jail from a backup archive."""
+    logger = ctx.parent.logger
+    zfs: iocage.ZFS.ZFS = ctx.parent.zfs
+    host: iocage.Host.HostGenerator = ctx.parent.host
```

Comment:
does this mean you can import from a *remote* host?

Reply:
Yes, of course! That's the whole purpose of a backup export 🙊